logdetective/server/server.py

```diff
@@ -549,13 +549,16 @@ async def retrieve_and_preprocess_koji_logs(
         if zipinfo.filename.endswith("task_failed.log"):
             # The koji logs store this file in two places: 1) in the
             # directory with the failed architecture and 2) in the parent
-            # directory. We actually want to ignore the one in the parent
-            # directory, since the rest of the information is in the
-            # specific task directory.
+            # directory. Most of the time, we want to ignore the one in the
+            # parent directory, since the rest of the information is in the
+            # specific task directory. However, there are some situations
+            # where non-build failures (such as "Target build already exists")
+            # may be presented only at the top level.
```

Comment:
Now that I read this, we should think about providing guidance to these build problems as well in the future.